.github/workflows/test.yaml

```diff
@@ -712,25 +771,26 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - uses: pnpm/action-setup@v2.2.4
+        with:
+          version: 7
+
       - uses: actions/setup-node@v3
         with:
-          node-version: 14
-          cache: 'yarn'
-          # Only cache the root node_modules
-          cache-dependency-path: 'yarn.lock'
+          node-version: 16
+          cache: ${{ contains(matrix.platform, 'codesandbox') && 'npm' || 'pnpm' }}
+          cache-dependency-path: ${{ github.job }}/${{ matrix.platform }}/${{ contains(matrix.platform, 'codesandbox') && 'package-lock.json' || 'pnpm-lock.yaml' }}
 
       - name: Define Client Engine Type to test
         run: echo "PRISMA_CLIENT_ENGINE_TYPE=${{ matrix.clientEngine }}" >> $GITHUB_ENV
 
       - name: Install Dependencies
-        run: yarn install
+        run: ${{ contains(matrix.platform, 'codesandbox') && 'npm install' || 'pnpm install' }}
```

Comment:
pnpm does not work on codesandbox?

Reply:
Codesandbox automatically install npm or yarn but not pnpm